code/game/objects/items/cards_ids.dm

```diff
@@ -369,10 +369,8 @@ update_label()
 
 		var/popup_input = alert(user, "Action", "Agent ID", "Show", "Forge", "Change Account ID")
 		if(popup_input == "Forge")
-			var/t = copytext(sanitize(input(user, "What name would you like to put on this card?", "Agent card name", registered_name ? registered_name : (ishuman(user) ? user.real_name : user.name))as text | null),1,26)
-			if(!t || t == "Unknown" || t == "floor" || t == "wall" || t == "r-wall") //Same as mob/dead/new_player/prefrences.dm
-				if (t)
-					alert("Invalid name.")
+			var/t = copytext(sanitize_name(input(user, "What name would you like to put on this card?", "Agent card name", registered_name ? registered_name : (ishuman(user) ? user.real_name : user.name))as text | null),1,26)

```

Comment:
I decided to not put the ic filter check on the job because it isn't as referred to as much, but maybe i should?